common/openapi.yml

```diff
@@ -258,8 +258,78 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/policy/v1alpha1/region/{objectId}/evaluate:
+    parameters:
+      - $ref: '#/components/parameters/TpsObjectId'
+      - $ref: '#/components/parameters/Datacenter'
+      - $ref: '#/components/parameters/Platform'
+    get:
+      summary: Evaluate whether a datacenter is available inside of a region constraint
+      operationId: validateDatacenterAllowed
+      tags: [Tps]
+      responses:
+        '204':
+          description: Datacenter evaluated successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorReport'
```

Comment:
That doesn't make sense. 204 means no content, but you are supplying content.

Reply:
yeah, that's the problem I was trying to describe with openapi. If we use the global exception handler then we don't need this.